official/keras_application_models/benchmark_main.py

```diff
@@ -41,13 +41,14 @@
     "densenet121": tf.keras.applications.DenseNet121,
     "densenet169": tf.keras.applications.DenseNet169,
     "densenet201": tf.keras.applications.DenseNet201,
-    # TODO (b/80431378)
+    # TODO(b/80431378)
     # "nasnetlarge": tf.keras.applications.NASNetLarge,
     # "nasnetmobile": tf.keras.applications.NASNetMobile,
 }
 
 
-def main(_):
+def run_keras_model_benchmark(_):
```

Comment:
You can choose to use the input param here which is the flag object, and that will enable the value injection during test. This can be done in a later PR.